src/main/java/com/hotline/pageobject/pages/ApplePhonesPage.java

```diff
@@ -0,0 +1,39 @@
+package com.hotline.pageobject.pages;
+
+import com.hotline.pageobject.modules.HeaderModule;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class ApplePhonesPage extends HeaderModule {
+    private List<WebElement> phoneNameList;
+    private WebElement mainSort;
+    private WebElement mainSortBox;
+    private List<WebElement> phonePriceList;
+
+    ApplePhonesPage(WebDriver driver) {
+        super(driver);
+        phoneNameList = driver.findElements(By.xpath("//*[@class='item-info']/p/a"));
```

Comment:
Move initialization of elements to respective get methods (let's try to implement lazy initialization from now on)